t/venom/test_suites/fingerbank_proxy/fingerbank_connectivity/fingerbank_connectivity.yml

```diff
@@ -0,0 +1,35 @@
+name: FingerbankConnectivity
```

Comment:
It's not possible to use PacketFence API to validate account info and connectivity to the API ?

Reply:
I wanted to stay as close to the Fingerbank library as possible in order to prevent any caching mechanism we have in PF and the layers of the Fingerbank librarires

Not that we can't, it's just that I found it closer to want we really want to test here while ensuring we have the least possible chances of hitting a cache (keeping in mind we may add other caching mechanisms later on and forget to update this suite)

Reply:
I understand the idea. If we stay like this, I wonder if we can say we really test Fingerbank like a user will do.

I mean if we break API call in PacketFence, are we be able to detect it ? But I'm 99% sure API call is calling directly piece of code you add here.

Reply:
I'm not talking about the API code in PF being broken but more that we can hit a layer of caching inside PF that will prevent to detect the failure properly if the proxy integration stops working

Making it further away from the actual collector/perl lib code gives us more chances of hitting a cache